crates/api_models/src/enums.rs

```diff
@@ -131,6 +131,7 @@ pub enum Connector {
     // Tsys,
```

Comment:
Nit: Can we remove this line, maybe in a separate PR?

Reply:
sure